mobile_config/migrations/2_registered_keys.sql

```diff
@@ -0,0 +1,15 @@
+create type key_type as enum (
+    'administrator',
+    'packet_router'
+    'oracle',
```

Comment:
postgres errors on this statement, can't have ending `,` .

Reply:
spoiled by rust; fixed [3e2ca43](https://github.com/helium/oracles/pull/419/commits/3e2ca43b54d5f0aaeaef51ca36ecb1aec3299408)